### PR TITLE
[Translate ru] Fundamentals -> index

### PR DESCRIPTION
**Pull request by DmitrySkripunov**
What's changed, or what was fixed?
- translate on Russian

### CODE REVIEW

File: src/data/_contributors.yaml [hunk 1]
```diff
@@ -265,17 +265,6 @@ samchen:
   description:
     en: Sam is a Front End Developer
 
-hongchanchoi:
```
Comment (agektmr) [inline src/data/_contributors.yaml]: Why did you delete honchangchoi? If this is an unexpected change, please revert.

Comment (DmitrySkripunov) [reply src/data/_contributors.yaml]: Wow, sorry, I don't know how it happen! 
I did commit with revert,  look at it, please.

**Comment by DmitrySkripunov** [comment]
@petele I made fix, could you loot at this file, please?

**Comment by sotayamashita** [comment]
@petele Please check it out

**Comment by ilyaspiridonov** [comment]
@agektmr Russian translation reviewed, although the PR seems to be outdated, as there's no longer an index.md file (superseded with _index.yaml)
